<launch>

  <arg name="image_topic" default="/realsense_face/color/image_raw" />
  <arg name="publish_viz" default="true" />

  <node pkg="openface2_bridge" name="headpose" type="headpose" output="screen">
      <param name="image_topic" value="$(arg image_topic)" type="str"/>
      <param name="publish_viz" value="$(arg publish_viz)" type="bool"/>
  </node>

</launch>
